cmd/regex_format.go

```diff
@@ -21,8 +23,13 @@ import (
 
 // formatCmd represents the generate command
 var formatCmd = createFormatCommand()
-var preprocessorStartRegex = regexp.MustCompile(`^##!>`)
-var preprocessorEndRegex = regexp.MustCompile(`^##!<`)
+var blockStartRegex = regexp.MustCompile(`^##!>\s*(assemble|cmdline)\s*(\S+)?`)
+var blockEndRegex = regexp.MustCompile(`^##!<`)
+var includeRegex = regexp.MustCompile(parser.IncludePattern)
+var definitionRegex = regexp.MustCompile(parser.DefinitionPattern)
+var prefixRegex = regexp.MustCompile(parser.PrefixPattern)
+var suffixRegex = regexp.MustCompile(parser.SuffixPattern)
+var flagsRegex = regexp.MustCompile(parser.FlagsPattern)
```

Comment:
I wonder if having a `map[string]regexp` would be more elegant here 🤔 
Example:
```
var regex = map[string]regexp{
  "blockStart": regexp.MustCompile....
```
Don't know if it works also.

Reply:
It would look more elegant w.r.t. the definition, I agree. However, I don't like using magic constants to get the entries from the map, so I would then either define constants or methods to encapsulate that information. That seems like a lot of unnecessary overhead to me.